data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -1777,6 +1777,19 @@
   ]
 },{
   "type" : "recipe",
+  "result": "lye_powder",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_CHEMICALS",
+  "skill_used": "cooking",
+  "difficulty": 3,
+  "time": 90000,
+  "autolearn": true,
+  "batch_time_factors": [ 80, 4 ],
+  "qualities": [ { "id": "BOIL", "level": 1 } ],
+  "tools": [ [ [ "water_boiling_heat", 3, "LIST" ] ] ],
```

Comment:
3 is just enough energy to bring a cup of water to boiling temperature. But the recipe involves boiling out 20 liters of liquid.

Reply:
Thanks for pointing that out!